.husky/pre-push

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npm test -- --runInBand --watchAll=false --bail --ci
```

Comment:
we don't actually need to run tests in band this was just for when we were testing the database in the email product